observer/probers/tls/tls.go

```diff
@@ -123,7 +132,7 @@ func (p TLSProbe) probeExpired(timeout time.Duration) bool {
 	}
 
 	root := peers[len(peers)-1].Issuer
-	if root.Organization[0] != p.rootOrg || root.CommonName != p.rootCN {
+	if err = p.checkRoot(root.Organization[0], root.CommonName); err != nil {
```

Comment:
While this is definitely correct, this is a style that we try to avoid.
```suggestion
	err = p.checkRoot(root.Organization[0], root.CommonName) 
	if err != nil {
```

Reply:
(And for future readers, this style choice is documented here: https://github.com/letsencrypt/boulder/blob/main/docs/CONTRIBUTING.md#error-handling)